lightningd/status.c

```diff
@@ -99,5 +108,10 @@ void status_failed(u16 code, const char *fmt, ...)
 	status_send_with_hdr(code, str, strlen(str));
 	va_end(ap);
 
+	/* Don't let it take forever. */
+	alarm(10);
```

Comment:
Will the alarm cause the process to exit or will it interrupt the write? If it exits, what status code will it give?